Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/rapidantigen/RapidAntigenProcessor.kt

```diff
@@ -41,6 +43,10 @@ class RapidAntigenProcessor @Inject constructor(
 
         val testResult = registrationData.testResult.validOrThrow()
 
+        if (testResult == PCR_OR_RAT_PENDING) {
```

Comment:
PCR_OR_RAT_PENDING || RA_PENDING

PCR has one pending state, RA has two.

Reply:
thank you